en/companion_computer/holybro_pixhawk_jetson_baseboard.md

```diff
@@ -253,7 +257,7 @@ Dimensions
 
 ![Jetson Pinouts](../../assets/companion_computer/holybro_pixhawk_jetson_baseboard/jetson_pinout.png)
 
-::: details Power ports
+::: details Pixhawk ports
 
 ### Power 1 (Main), Power 2 Ports
```

Comment:
FYI these ports were described as the power ports, but they are the Pixhawk power ports and don't need to be pulled out separately.